src/components/Workflow/PlanSheet.tsx

```diff
@@ -740,6 +739,28 @@ export function PlanSheet(props: IProps) {
                       className: 'book ' + calcClassName,
                       dataEditor: bookEditor,
                     }
+                  : cellIndex === refCol
+                  ? {
+                      value:
+                        passageTypeFromRef(e as string, inlinePassages) !==
+                        PassageTypeEnum.PASSAGE ? (
+                          <RefRender
+                            value={e as string}
+                            flat={inlinePassages}
+                          />
+                        ) : (
+                          e
+                        ),
+                      readOnly:
+                        readonly ||
+                        passageTypeFromRef(e as string, inlinePassages) !==
+                          PassageTypeEnum.PASSAGE,
+                      className:
```

Comment:
Do we still need this special classname?

Reply:
I think if the user enters a bad reference, we show the error with this class name. It helps make sure we can post to Paratext.

Reply:
We could probably simplify the test since we know we are in the reference column.

Reply:
I simplified it.